.github/pull_request_template.md

```diff
@@ -1,20 +1,53 @@
-## Description of the change
+## What are the changes introduced in this PR?
 
-< Description >
+Write a brief explainer on your code changes.
 
-## Checklists
+## What is the related Linear task?
 
-### Development
+Resolves INT-XXX
+
+## Please explain the objectives of your changes below
+
+Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here
+
+### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?
+
+N/A
+
+### Any new dependencies introduced with this change?
+
+N/A
+
+### Any new generic utility introduced or modified. Please explain the changes.
+
+N/A
+
+### Any technical or performance related pointers to consider with the change?
+
+N/A
+
+<hr>
+
+### Developer checklist
 
 - [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
-- [ ] The code changed/added as part of this pull request has been covered with tests
-- [ ] All tests related to the changed code pass in development
 
-### Code review
+- [ ] **No breaking changes are being introduced.**
+
+- [ ] All related docs linked with the PR?
+
+- [ ] All changes manually tested?
+
+- [ ] Any documentation changes needed with this change?
+
+- [ ] Is the PR limited to 10 file changes?
+
+- [ ] Is the PR limited to one linear task?
+
+- [ ] Are relevant unit and component test-cases added?
+
+### Reviewer checklist
+
+- [ ] Is the type of change in the PR title appropriate as per the changes?
 
-- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
-- [ ] Changes have been reviewed by at least one other engineer
-- [ ] Issue from task tracker has a link to this pull request
+- [ ] Verified that there are no credentials or confidential data exposed with the changes.
```

Comment:
is this valid here?

Reply:
Not really sure 😅 @koladilip any advice here ?

Reply:
Then we will have it 👍🏼